### ISSUE TITLE
This error happens when I open the Dungeon and raid tool

### ISSUE THREAD

**Issue by ioprophet** [Azilroka/AddOnSkins#20]
![RaiderIO-Addonskinserror](https://user-images.githubusercontent.com/3488663/139908421-3be2ec39-9ed2-484e-b92a-bd3b510b0d8d.jpg)

**Comment by Rykka-Stormheart** [comment]
Issue confirmed fixed by changing the following from this:
`function AS:RaiderIO()
	if _G.RaiderIO_CustomDropDownListMenuBackdrop then
		AS:SkinTooltip(_G.RaiderIO_CustomDropDownListMenuBackdrop)
	end

	_G.PVEFrame:HookScript("OnShow", function(self)
		if not _G.RaiderIOProfileTooltip.IsSkinned then
			AS:SkinFrame(_G.RaiderIOProfileTooltip)

			_G.RaiderIOProfileTooltip.IsSkinned = true
		end
	end)
end`

to the following:
`function AS:RaiderIO()
	if _G.RaiderIO_CustomDropDownListMenuBackdrop then
		AS:SkinTooltip(_G.RaiderIO_CustomDropDownListMenuBackdrop)
	end

	_G.PVEFrame:HookScript("OnShow", function(self)
		if not _G.RaiderIO_ProfileTooltip.IsSkinned then
			AS:SkinFrame(_G.RaiderIO_ProfileTooltip)

			_G.RaiderIO_ProfileTooltip.IsSkinned = true
		end
	end)
end`
in file `RaiderIO.lua`, inside `AddOnSkins\Skins\Mainline\AddOns`

TL;DR:
`RaiderIOProfileTooltip` needs to be `RaiderIO_ProfileTooltip`
/fstack shows the correct frame name:
![](https://i.imgur.com/FS3Mh4p.png)
